initramfs/init.sh

```diff
@@ -155,9 +176,9 @@ parse_cmdline()
 			_prefix="${_root%%=*}"
 
 			if [ "${_prefix}" = "UUID" ] || \
-			   [ "${_prefix}" = "PARTUUID" ] || \
-			   [ "${_prefix}" = "LABEL" ] || \
-			   [ "${_prefix}" = "PARTLABEL" ]; then
+				[ "${_prefix}" = "PARTUUID" ] || \
+				[ "${_prefix}" = "LABEL" ] || \
+				[ "${_prefix}" = "PARTLABEL" ]; then
```

Comment:
:-1: 
I purposely aligned the start oft he test for readability. Now the first [ "${_prefix}" no longer matches up, making it a little harder to read

Reply:
whatever >:P